javascript/javascript1/week1/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,12 @@
+const dogYearOfBirth = 2015;
+const dogYearFuture = 2027;
+let dogYear;
+let shouldShowResultInDogYears = true;
+if(shouldShowResultInDogYears==true){
```

Comment:
Remember, in javascript we like to use `===` rather than `==` when comparing two values 😃